modules/index.d.ts

```diff
@@ -291,6 +292,7 @@ export interface BaseWidget {
   hideOperator?: boolean,
   //obsolete:
   validateValue?: ValidateValue,
+  jsonLogic?: string | JsonLogicFormatValue,
```

Comment:
Thanks for PR. Can you please move it above `//obsolete`?

Reply:
Now moved.